index-handlers/src/main/java/no/sikt/nva/nvi/index/model/document/PublicationDetails.java

```diff
@@ -2,18 +2,18 @@
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
 import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import java.util.List;
 
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 @JsonSerialize
-public record PublicationDetails(@JsonProperty("id") String id,
```

Comment:
The JsonProperty-annotations are not necessary here since the field name is the same